reporter/src/main/kotlin/reporters/AbstractNoticeReporter.kt

```diff
@@ -57,6 +57,7 @@ abstract class AbstractNoticeReporter : Reporter() {
     class PostProcessor(ortResult: OrtResult, noticeReport: NoticeReport) : ScriptRunner() {
         override val preface = """
             import com.here.ort.model.OrtResult
+            import com.here.ort.model.spdx.SpdxLicense
```

Comment:
Can you also add `com.here.ort.model.config.LicenseMapping`, please? Btw, this class should probably also be moved to the spdx package, maybe you want to do that, too :)

Reply:
Done.

Reply:
Thanks.